web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -636,6 +656,111 @@ export function AppAccessSpecSection({
   );
 }
 
+export function DatabaseAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<DatabaseAccessSpec>) {
+  return (
+    <>
+      <Box mb={3}>
+        <Text typography="body3" mb={1}>
+          Labels
+        </Text>
+        <LabelsInput
+          disableBtns={isProcessing}
+          labels={value.labels}
+          setLabels={labels => onChange?.({ ...value, labels })}
+        />
+      </Box>
+      <FieldSelectCreatable
+        isMulti
+        label="Database Names"
+        toolTipContent={
+          <>
+            List of database names that this role is allowed to connect to.
```

Comment:
For some reason the dot at the end of this line is not visible in the tooltip for me.
<img width="540" alt="no-dot" src="https://github.com/user-attachments/assets/2bcbabf3-f6d7-496b-b0db-4655a543a5b8">